build.gradle

```diff
@@ -235,7 +235,7 @@ dependencies {
 
     compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.0.0'
 
-    compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '2.0.9'
+    compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '2.0.11-RC1'
```

Comment:
kindly reminder we need to use the Release version rather than the RC one.

Reply:
@iguannago are you happy for it to be released, as it's an addition.